.github/workflows/cleanup_pip_cache.yml

```diff
@@ -0,0 +1,31 @@
+name: Cleanup PIP caches
+on:
+  schedule:
+    # at 00:00 on the 1st day of every month
+    - cron: '0 0 1 * *'
+
+jobs:
+  Cleanup_PIP_Caches:
+    runs-on: aks-linux-2-cores
+    container:
+      image: openvinogithubactions.azurecr.io/dockerhub/ubuntu:20.04
+      volumes:
+        - /mount/caches:/mount/caches
+    env:
+      PIP_CACHE_PATH: /mount/caches/pip
+
+    steps:
+      - name: Pre-Collecting Cache Info
+        run: |
+          echo "Cache info: "
+          du -h -d2 ${PIP_CACHE_PATH}
+
+      - name: Cleanup cache
+        run: |
+          echo "Removing pip caches"
+          [ ! -z "${PIP_CACHE_PATH}" ] && rm -rv ${PIP_CACHE_PATH}/*
```

Comment:
should we drop caches only for non-latest pip versions?

Reply:
The problem that  different python installation may use different pip versions

Reply:
agree with Alina to remove files based on last access time

Reply:
done